demo/web/index.html

```diff
@@ -9,69 +9,80 @@
+              },
+            ],
+            porcupineKeywordCallback,
+            porcupineErrorCallback
+          );
 
-        ppnEn.onmessage = keywordDetection;
-        ppnEs.onmessage = keywordDetection;
-        ppnFr.onmessage = keywordDetection;
-        ppnDe.onmessage = keywordDetection;
+          let ppnDe = await PorcupineWebDeWorker.PorcupineWorkerFactory.create(
+            accessKey,
+            {
+              builtin: "Ananas",
+              sensitivity: 0.7,
+            },
+            porcupineKeywordCallback,
+            porcupineErrorCallback
+          );
+          let ppnEs = await PorcupineWebEsWorker.PorcupineWorkerFactory.create(
+            accessKey,
+            {
+              builtin: "Manzana",
+              sensitivity: 0.6,
+            },
+            porcupineKeywordCallback,
+            porcupineErrorCallback
+          );
+          let ppnFr = await PorcupineWebFrWorker.PorcupineWorkerFactory.create(
+            accessKey,
+            ["Framboise"],
```

Comment:
Why is this one different from the other languages?

Reply:
there are different ways to create the worker and pass keywords to the engine. I think the idea here was to show different possibilities (I got this part from the old demo written by David)

Reply:
Okay, I figured, just wanted to make sure it wasn't a mistake.